ModularTegustation/tegu_items/lc13unique_items.dm

```diff
@@ -1,3 +1,33 @@
+	//Admin Quick Leveler
+/obj/item/lc13_attribute_tester
+	name = "attribute injector"
+	desc = "A fluid used to drastically change a employee for tests. Use in hand to activate."
+	icon = 'ModularTegustation/Teguicons/teguitems.dmi'
+	icon_state = "oddity7"
+
+/obj/item/lc13_attribute_tester/attack_self(mob/living/carbon/human/user)
+	to_chat(user, "<span class='nicegreen'>You suddenly feel different.</span>")
+	user.adjust_all_attribute_levels(100)
+	qdel(src)
+
+/obj/item/lc13_easygift_tester
+	name = "gift extractor"
+	desc = "Unpopular due to its excessive energy use, this device extracts gifts from a entity on demand."
+	icon = 'icons/obj/items_and_weapons.dmi'
+	icon_state = "hammeroff"
+
+/obj/item/lc13_easygift_tester/attack(mob/living/simple_animal/hostile/abnormality/M, mob/living/carbon/human/user)
```

Comment:
You forgot to add a check for if the mob is an abnormality type at all.

Reply:
I added a isabnormality(M)